diff --git a/ios/ScreenOrientationReactDelegateHandler.swift b/ios/ScreenOrientationReactDelegateHandler.swift
index daf743717bcf558200f39e0120dc5147ed2b9d08..2cb7ad28f84718a34c0c501bd30b633d8a9ffb6e 100644
--- a/ios/ScreenOrientationReactDelegateHandler.swift
+++ b/ios/ScreenOrientationReactDelegateHandler.swift
@@ -4,7 +4,7 @@ import ExpoModulesCore
 
 @objc(EXScreenOrientationReactDelegateHandler)
 public class ScreenOrientationReactDelegateHandler: ExpoReactDelegateHandler {
-  public override func createRootViewController() -> UIViewController? {
+  public func createRootViewController() -> UIViewController? {
     return ScreenOrientationViewController(defaultScreenOrientationFromPlist: ())
   }
 }
