diff --git a/android/src/main/java/com/doublesymmetry/trackplayer/module/MusicModule.kt b/android/src/main/java/com/doublesymmetry/trackplayer/module/MusicModule.kt
--- a/android/src/main/java/com/doublesymmetry/trackplayer/module/MusicModule.kt
+++ b/android/src/main/java/com/doublesymmetry/trackplayer/module/MusicModule.kt
@@ -545,7 +545,10 @@ class MusicModule(reactContext: ReactApplicationContext) :
         if (verifyServiceBoundOrReject(callback)) return@launch

         if (index >= 0 && index < musicService.tracks.size) {
-            callback.resolve(Arguments.fromBundle(musicService.tracks[index].originalItem))
+            musicService.tracks[index].originalItem?.let {
+                callback.resolve(Arguments.fromBundle(it))
+            } ?: callback.resolve(null)
         } else {
             callback.resolve(null)
         }
@@ -585,7 +588,10 @@ class MusicModule(reactContext: ReactApplicationContext) :
-        callback.resolve(
-            if (musicService.tracks.isEmpty()) null
-            else Arguments.fromBundle(
-                musicService.tracks[musicService.getCurrentTrackIndex()].originalItem
-            )
-        )
+        if (musicService.tracks.isEmpty()) {
+            callback.resolve(null)
+        } else {
+            musicService.tracks[musicService.getCurrentTrackIndex()].originalItem?.let {
+                callback.resolve(Arguments.fromBundle(it))
+            } ?: callback.resolve(null)
+        }
     }
