diff --git a/node_modules/react-native/scripts/xcode/ccache-clang++.sh b/node_modules/react-native/scripts/xcode/ccache-clang++.sh
index f699d9a..e6e1be0 100755
--- a/node_modules/react-native/scripts/xcode/ccache-clang++.sh
+++ b/node_modules/react-native/scripts/xcode/ccache-clang++.sh
@@ -11,4 +11,6 @@ REACT_NATIVE_CCACHE_CONFIGPATH=$SCRIPT_DIR/ccache.conf
 # Provide our config file if none is already provided
 export CCACHE_CONFIGPATH="${CCACHE_CONFIGPATH:-$REACT_NATIVE_CCACHE_CONFIGPATH}"
 
+export PATH="$PATH:/usr/local/bin:/opt/homebrew/bin/"
+
 exec ccache clang++ "$@"
diff --git a/node_modules/react-native/scripts/xcode/ccache-clang.sh b/node_modules/react-native/scripts/xcode/ccache-clang.sh
index 5ed8664..f85d3b2 100755
--- a/node_modules/react-native/scripts/xcode/ccache-clang.sh
+++ b/node_modules/react-native/scripts/xcode/ccache-clang.sh
@@ -11,4 +11,6 @@ REACT_NATIVE_CCACHE_CONFIGPATH=$SCRIPT_DIR/ccache.conf
 # Provide our config file if none is already provided
 export CCACHE_CONFIGPATH="${CCACHE_CONFIGPATH:-$REACT_NATIVE_CCACHE_CONFIGPATH}"
 
+export PATH="$PATH:/usr/local/bin:/opt/homebrew/bin/"
+
 exec ccache clang "$@"
